osu.Game.Tournament.Tests/TournamentTestBrowser.cs

```diff
@@ -19,6 +19,9 @@ protected override void LoadComplete()
                 Depth = 10
             }, AddInternal);
 
+            MenuCursorContainer.Cursor.AlwaysPresent = true;
```

Comment:
would it not make sense to leave this in the base implementation if it's done in both places?

Reply:
Right, I didn't consider that. I moved back to the base implementation